include/megaapi.h

```diff
@@ -14132,6 +14133,18 @@ class MegaApi
         char *getBlockedPath();
 #endif
 
+        /**
+         * @brief Start backup for the given folder, to "My Backups" remote destination
+         *
+         * The backup's folder name is optional. If not provided, it will take the name of the leaf folder of
+         * the local path. In example, for "/home/user/Documents", it will become "Documents".
+         *
```

Comment:
Missing documentation about the request params. Please, use doc from other interfaces as reference. It should indicate the valid params, both at any callback and when it returns API_OK. Also, it should list some of the non-obvious errors the request may return.